src/OpenSage.Game/Content/ModelLoader.cs

```diff
@@ -105,7 +105,7 @@ protected override Model LoadEntry(FileSystemEntry entry, ContentManager content
                     subObjects.Add(new ModelSubObject(w3dSubObject.Name, bone, mesh));
                 }
             }
-            else
+            else if (w3dMeshes.Count > 0)
```

Comment:
I guess this fixes a crash?

Reply:
Yes, the one you found in CHI01. Now that we can load it from the menu, it seemed somewhat related :)